### PR TITLE
QEMU ubuntu fix & update

### PR DESCRIPTION
**Pull request by lukaszstolarczuk**
Update and fix QEMU Ubuntu

// preview nightly: https://github.com/lukaszstolarczuk/unified-memory-framework/actions/runs/12068436829

### CODE REVIEW

File: .github/workflows/nightly.yml [hunk 1]
```diff
@@ -191,4 +191,4 @@ jobs:
     uses: ./.github/workflows/reusable_qemu.yml
     with:
       short_run: false
-      os: "['ubuntu-22.04', 'ubuntu-24.04']"
+      os: "['ubuntu-22.04', 'ubuntu-24.04', 'ubuntu-24.10']"
```
[the remark this answers is not in the document]
Comment (lukaszstolarczuk) [reply .github/workflows/nightly.yml:194]: why not? we can test the latest Ubuntu with zero cost

[the remark this answers is not in the document]
Comment (lplewa) [reply .github/workflows/nightly.yml:194]: 0 cost = 50% more builds? ;)

[the remark this answers is not in the document]
Comment (bratpiorka) [reply .github/workflows/nightly.yml:194]: @plewa this is the nightly workflow

[the remark this answers is not in the document]
Comment (KFilipek) [reply .github/workflows/nightly.yml:196]: I'm not a fan of commenting everything-everywhere. This comment doesn't provide any value unlike the `ubuntu-24.10`.

**Review by PatKamin — APPROVED** [review]

**Review by bratpiorka — APPROVED** [review]

**Pull request merged**